ts/parts/PlotLineOrBand.ts

```diff
@@ -82,7 +82,7 @@ declare global {
             events?: any;
             id?: string;
             label?: AxisPlotLinesLabelOptions;
-            value?: number;
+            value: number;
```

Comment:
This has to be kept optional, because it is not describing a code-side interface, but an options interface from our API options doclets. These properties are always optional by nature, because a user can choose, which should be set and which not.

Reply:
👍